### PR TITLE
Ros2 devel

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: panther_gazebo/panther_gazebo/launch/panther.launch.py [hunk 1]
```diff
@@ -10,35 +10,29 @@
 def generate_launch_description():
   panther_gazebo = get_package_share_directory('panther_gazebo')
   panther_description = get_package_share_directory('panther_description')
-  world = LaunchConfiguration('world', default=os.path.join(panther_description, 'worlds', 'willowgarage.world'))
-  panther_description_config_path = os.path.join(panther_description, 'models', 'panther', 'config')
+  panther_description_config_path = os.path.join(panther_description, 'config')
 
-  panther_common_props_path = LaunchConfiguration('panther_common_props_path', default=os.path.join(panther_description_config_path, 'panther_common.yaml'))
-  wheel_props_path = LaunchConfiguration('wheel_props_path', default=os.path.join(panther_description_config_path, 'WH01.yaml'))
+  wheel_config_path = LaunchConfiguration('wheel_config_path', default=os.path.join(panther_description_config_path, 'WH04.yaml'))
```
Comment (KmakD) [inline panther_gazebo/panther_gazebo/launch/panther.launch.py]:
```suggestion
  wheel_config_path = LaunchConfiguration('wheel_config_path', default=os.path.join(panther_description_config_path, 'WH01.yaml'))
```
I guess WH01 should be default

File: panther_gazebo/panther_gazebo/package.xml [hunk 2]
```diff
@@ -3,21 +3,28 @@
...
-  <build_depend>panther_description</build_depend>
+  <description>Gazebo-classic simualiton configuration for Husarion Panther robot</description>
+  <license>Apache License 2.0</license>
+
+  <author email="krzysztof.wojciechowski@husarion.com">Krzysztof Wojciechowski</author>
+  <maintainer email="support@husarion.com">Husarion</maintainer>
+
+  <url type="website">https://husarion.com</url>
+  <url type="bugtracker">https://github.com/husarion/panther_description/issues</url>
+  <url type="repository">https://github.com/husarion/panther_description/tree/ros2</url>
```
Comment (KmakD) [inline panther_gazebo/panther_gazebo/package.xml]: In other repos "repository" is before "bugtracker"

File: panther_gazebo/panther_gazebo/package.xml [hunk 3]
```diff
@@ -3,21 +3,28 @@
...
 
-  <exec_depend>controller_manager</exec_depend>
-  <exec_depend>joint_state_controller</exec_depend>
-  <exec_depend>joint_state_publisher</exec_depend>
   <exec_depend>gazebo_ros</exec_depend>
-  <exec_depend>robot_state_publisher</exec_depend>
-  <exec_depend>panther_description</exec_depend>
+  <exec_depend>gazebo_plugins</exec_depend>
```
Comment (KmakD) [inline panther_gazebo/panther_gazebo/package.xml]: Packages not in alphabetical order

**Review by KmakD — CHANGES_REQUESTED** [review]
I don't know if all of this apply to this PR but there are few things:
- Python scripts does not follow coding style 
- Includes are not in alphabetical order
- Dockerfile is a mess

File: panther_gazebo/panther_gazebo/launch/panther.launch.py (hunk 1 above)
Comment (Kotochleb) [reply panther_gazebo/panther_gazebo/launch/panther.launch.py]: Yeah. My bad

File: panther_gazebo/panther_gazebo/package.xml (hunk 2 above)
Comment (Kotochleb) [reply panther_gazebo/panther_gazebo/package.xml]: Done

File: panther_gazebo/panther_gazebo/package.xml (hunk 3 above)
Comment (Kotochleb) [reply panther_gazebo/panther_gazebo/package.xml]: Done

**Comment by Kotochleb** [comment]
> I don't know if all of this apply to this PR but there are few things:
> 
>     * Python scripts does not follow coding style
> 
>     * Includes are not in alphabetical order
> 
>     * Dockerfile is a mess

We will move all those changes to new task and new PR in the next sprint

File: panther_gazebo/panther_gazebo/package.xml [hunk 4]
```diff
@@ -3,21 +3,26 @@
...
-  <exec_depend>xacro</exec_depend>
+  <author email="krzysztof.wojciechowski@husarion.com">Krzysztof Wojciechowski</author>
+  <maintainer email="support@husarion.com">Husarion</maintainer>
+
+  <url type="website">https://husarion.com</url>
+  <url type="repository">https://github.com/husarion/panther_description/tree/ros2</url>
+  <url type="bugtracker">https://github.com/husarion/panther_description/issues</url>
+
+  <depend>gazebo_plugins</depend>
+  <depend>gazebo_ros</depend>depend
```
Comment (KmakD) [inline panther_gazebo/panther_gazebo/package.xml]:
```suggestion
  <depend>gazebo_ros</depend>
```

**Review by KmakD — CHANGES_REQUESTED** [review]

**Review by KmakD — APPROVED** [review]

**Pull request merged**